torch/_dynamo/config.py

```diff
@@ -68,6 +69,18 @@
 # see [Note - on the state of mark_dynamic]
 assume_static_by_default = False
 
+
+class DYNAMIC_SHAPE_STRATEGY(Enum):
+    OFF = 1
+    # Find all tensors in frame and mark them all dynamic if they fail
```

Comment:
*Every* tensor in the frame? That seems a bit much! Don't you only want the ones that wobbled?

Reply:
Every *failed* tensor in the frame. I accidentally a word. However, the enum name is right!